docs/dev_guide/index.rst

```diff
@@ -16,26 +16,54 @@ If you are thinking of contributing to sunpy please read the following carefully
 We currently recommend the :ref:`newcomers` as the place to start.
 This goes over the basics and has links to useful tutorials on git.
 
-.. toctree::
-   :maxdepth: 2
-
-   contents/newcomers
-   contents/code_standards
-   contents/documentation
-   contents/example_gallery
-   contents/tests
-   contents/pr_review_procedure
-   contents/api
-   contents/dependencies
-   contents/units_quantities
-   contents/new_objects
-   contents/extending_fido
-   contents/maintainer_workflow
-   contents/logger
-   contents/remote_data
-   contents/config
-   contents/funding
-   contents/ci_jobs
+.. panels::
+
+    Getting started
+    ^^^^^^^^^^^^^^^
+    .. toctree::
+      :maxdepth: 4
+
+      contents/newcomers
+
+    ---
+
+    Standards
+    ^^^^^^^^^
+    .. toctree::
+      :maxdepth: 1
+
+      contents/code_standards
+      contents/tests
+      contents/documentation
+      contents/example_gallery
+      contents/pr_review_procedure
+      contents/units_quantities
+
+    ---
+
+    Repo management
+    ^^^^^^^^^^^^^^^
+    .. toctree::
+      :maxdepth: 1
+
+      contents/dependencies
+      contents/funding
+      contents/maintainer_workflow
+      contents/ci_jobs
+
+    ---
+
+    API and extending sunpy
+    ^^^^^^^^^^^^^^^^^^^^^^^
+    .. toctree::
+      :maxdepth: 1
+
+      contents/api
```

Comment:
The title for this page is somewhat confusing in this context, I thought it would link to the API reference

Reply:
Renamed.